### PR TITLE
new pages for browse and search and various updates

### PR DESCRIPTION
**Pull request by saberraz**

### CODE REVIEW

File: www/pages/browse_data.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+permalink: /howto/browse_data
+---
+<!--- IMPORTANT: This permlink is referenced from InputApp -->
```
Comment (PeterPetrik) [inline www/pages/browse_data.md:5]: is it?

File: www/pages/search_data.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+permalink: /howto/search_data
+---
+<!--- IMPORTANT: This permlink is referenced from InputApp -->
```
Comment (PeterPetrik) [inline www/pages/search_data.md:5]: is it?

File: www/pages/working_with_nonspatial_data.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+permalink: /howto/working_with_nonspatial_data
+---
+<!--- IMPORTANT: This permlink is referenced from InputApp -->
```
Comment (PeterPetrik) [inline www/pages/working_with_nonspatial_data.md:5]: is it?

File: www/pages/browse_data.md (hunk 1 above)
Comment (saberraz) [reply www/pages/browse_data.md:5]: No...I did not add the comment!

File: www/pages/search_data.md (hunk 2 above)
Comment (saberraz) [reply www/pages/search_data.md:5]: No...I did not add the comment!

File: www/pages/working_with_nonspatial_data.md (hunk 3 above)
Comment (saberraz) [reply www/pages/working_with_nonspatial_data.md:5]: No...I did not add the comment!

**Comment by saberraz** [comment]
There is a general link from Input to help.inputapp.io, but afaik, there is no link to a specific page.

File: www/pages/search_data.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+You can search for the content of your attribute data from Input.
+
+# QGIS project configuration
+
+To be able to search the attribute table of a layer, you need to first configure your [QGIS project](https://docs.qgis.org/3.10/en/docs/user_manual/introduction/qgis_configuration.html?highlight=properties#data-sources-properties):
+
+- Open QGIS Desktop with you QGIS Project
+- In the Menu > Project > Properties click on "Data Sources"
+![browse_enable](../images/qgis_data_sources.png)
+- Under **Searchable** column, select the layers you'd like to search
```
Comment (wonder-sk) [inline www/pages/search_data.md:16]: Does it work like that? I though all layers are browsable by default...?

File: www/pages/search_data.md (hunk 2 above)
Comment (PeterPetrik) [reply www/pages/search_data.md:5]: we have automatic tests in inputapp that check that all pages that inputapp references exists. so this comment should be there only if this is such permlink page. see https://github.com/lutraconsulting/input/blob/c99b56c317f43dbb9b5ad69cd331d4d7c78cbbdd/app/test/testlinks.h#L50

File: www/pages/search_data.md (hunk 4 above)
Comment (saberraz) [reply www/pages/search_data.md:16]: It seems so!

**Pull request merged**